hrui/hrui.php

```diff
@@ -515,15 +515,17 @@ function hrui_civicrm_alterContent( &$content, $context, $tplName, &$object ) {
      * ( Normal Place of work =  $currentContractDetails->location )
     */
     $currentContractDetails = CRM_Hrjobcontract_BAO_HRJobContract::getCurrentContract($contact_id);
-    // $departmentsList contain current roles departments separated by comma
+    // $departmentsList contain current roles departments list separated by comma
     $departmentsList = null;
     if ($currentContractDetails)  {
-      $departmentsList = getDepartmentsList($currentContractDetails->contract_id);
+      $departmentsArray = CRM_Hrjobroles_BAO_HrJobRoles::getDepartmentsList($currentContractDetails->contract_id);
+      $departmentsList = implode(', ', array_values($departmentsArray));
```

Comment:
I think you didn't need array_values here, but it's ok, don't need to change it!